compiler/src/org.graalvm.compiler.replacements.test/src/org/graalvm/compiler/replacements/test/StringCompareToTest.java

```diff
@@ -100,15 +102,29 @@ public void testEqualString() {
     @Test
     @Ignore("GR-8748")
```

Comment:
I don't think we need this `@Ignore` any more right @mur47x111 ?

Reply:
My commit has removed this annotation and reenable the test.